utils/interface-validator.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+
+"""Schem validator for `interface.yaml`"""
+import yaml
+import glob
+import re 
+
+from enum import Enum
+from typing import List, Optional
+from pydantic import AnyHttpUrl, BaseModel, Field, ValidationError
+
+
+class StatusEnum(str, Enum):
+    draft = "draft"
```

Comment:
the enum member names should be uppercase, but I'm not sure if in this case it complicates your life too much?